src/documentation_glossary.jl

```diff
@@ -67,6 +67,7 @@ define!(:LaTeX, :Bigr, raw"\Bigr")
 define!(:LaTeX, :cos, raw"\cos")
 define!(:LaTeX, :def, raw"\coloneqq")
 define!(:LaTeX, :Cal, (letter) -> raw"\mathcal " * "$letter")
+define!(:LaTeX, :cos, raw"\cos")
```

Comment:
There is cos already three lines above?

Reply:
THen we can remove this line. Usually that is why I order them alphabetically and this one seems out of order if one takes small letters first.